### PR TITLE
added margin utility and updated views

### PR DESCRIPTION
**Pull request by twalton83**
Added an additional `push-down-2x` utility class and updated text sections referenced in issue below with said class.

Issue: #1441

### CODE REVIEW

File: app/assets/stylesheets/utilities.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .push-down {
   margin-bottom: 1rem;
 }
+
+.push-down-2x {
+  margin-bottom: 2rem;
+}
```
Comment (KevinMulhern) [inline app/assets/stylesheets/utilities.scss:7]: Can we have a newline between this and the push-down-3x rule please? just to keep it consistent with the other rules.

**Review by KevinMulhern — COMMENTED** [review]
Hi @twalton83, thanks for doing this. I have extremely minor suggestion but otherwise looks great 💯

File: app/assets/stylesheets/utilities.scss (hunk 1 above)
Comment (twalton83) [reply app/assets/stylesheets/utilities.scss:7]: Of course, I’ll add that later this afternoon.

**Review by KevinMulhern — APPROVED** [review]
Thanks @twalton83, great work 💯

**Pull request merged**